website_team/views/website_team_view.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
- <openerp>
+ <odoo>
     <data>
         <record id="view_users_form" model="ir.ui.view">
             <field name="name">res.users.form</field>
             <field name="model">res.users</field>
             <field name="type">form</field>
             <field name="inherit_id" ref="base.view_users_form"/>
             <field name="arch" type="xml">
-                <xpath expr="//div[@class='oe_title']//field[@name='active']" position="after">
+                <xpath expr="//field[@name='active']" position="after">
                     <field name="website_published"/>
```

Comment:
I don't see this button at the form
![image](https://user-images.githubusercontent.com/186131/62693085-53f67c00-b9c1-11e9-9e7f-5a55e7dcbb4d.png)